cpp/oneapi/dal/backend/primitives/reducer.cpp

```diff
@@ -0,0 +1,324 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <algorithm>
+#include <cmath>
+#include <exception>
+#include <limits>
```

Comment:
limits?

Reply:
Actually for initial values in some binary functors like min and max

Reply:
@KulikovNikita could you name the exact line where this include is needed?